src/Microsoft.VisualStudio.Editor.Razor/RazorLiveShareWorkspaceProvider.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
-    internal abstract class LiveShareWorkspaceProvider
+    public abstract class RazorLiveShareWorkspaceProvider
```

Comment:
Can we discuss this in more detail - I want to understand why we need to expose a type for them to implement. Right now we don't offer extensibility to other teams and I'd really like to avoid doing so. For instance there's nothing **Razor** about this type, it's just random plumbing.

Reply:
👍